src/workerd/jsg/modules-new-test.c++

```diff
@@ -0,0 +1,1889 @@
+// Copyright (c) 2017-2022 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+#include "jsg-test.h"
```

Comment:
Nit: use the more lightweight `#include <kj/test.h>` when not using Evaluator etc. (this was applied to existing code in 48f1af4e5bb9)